WordPress/Classes/Utility/Universal Links/Route.swift

```diff
@@ -58,3 +75,56 @@ extension Route {
         return path == route.path
     }
 }
+
+// MARK: - Tracking
+
+/// Where did the deep link originate?
+///
+enum DeepLinkSource: Equatable {
+    case link
+    case banner
+    case email(campaign: String)
+    case widget
+    case inApp(presenter: UIViewController?)
+
+    init?(string: String) {
```

Comment:
Nit: can we replace `string` with something else (e.g. `sourceName`)?

Reply:
Yep! 👍